workflow/controller/scope.go

```diff
@@ -68,13 +72,56 @@ func (s *wfScope) resolveParameter(v string) (string, error) {
 }
 
 func (s *wfScope) resolveArtifact(v string) (*wfv1.Artifact, error) {
-	val, err := s.resolveVar(v)
+
+	artRef := strings.SplitN(v, "/", 2)
```

Comment:
how many slashes could we have? what does 0/1/2/N slash mean?

Reply:
I think I'd like to be explicit and have a `subPath` field

Reply:
Yep I also favour the more explicit form for this as it gets a bit tricky/hacky resolving a mix of references in the artifact's "from" string 😄